### DISCUSSION TITLE
How do I disable the gray area on the right side of the screen?

### DISCUSSION THREAD

**Discussion by andrey-helldar** [gitextensions/gitextensions#11360] [Q&A] [answered]
Hey, everybody!

This appeared either after some update a few months ago, or after I pressed some hotkey. I don't know.

Tried to find both the button in the UI and the hotkey, and I'm not having any luck.

Can you please advise me how to turn off this gray area? It never shows any information, but you can move it with the mouse by pulling its border between the commit list.

![image](https://github.com/gitextensions/gitextensions/assets/10347617/b7c2d0cc-89ef-4157-83a3-0e0606d91da9)

**Comment by pmiossec** [comment, marked as answer]
Little compatibility bug when using old settings with newer version: play with "Commit info position" button in the toolbar and it should fix the issue.

![image](https://github.com/gitextensions/gitextensions/assets/460196/26fede25-42ca-47bc-9660-bf9fc7414a03)

**Reply by andrey-helldar** [reply]
Wow! It worked! I missed those buttons somehow))))

I was looking in the toolbar in the View tab and in the hotkey settings).

Just selected either one and the area instantly disappeared. Thanks!

![image](https://github.com/gitextensions/gitextensions/assets/10347617/b665ef61-7c9d-4548-8fe8-38de1248225c)

**Comment by gerhardol** [comment]
Also fixed in 4.2

**Reply by andrey-helldar** [reply]
![image](https://github.com/gitextensions/gitextensions/assets/10347617/3fe6b80c-70e1-4a7f-a816-d7cf24b94c93)

I took the screenshot on version 4.2.0.17591.

And I remembered when this area appeared - after updating from version 3 to version 4.
